### PR TITLE
Vendor SBOM schema

### PR DESCRIPTION
**Pull request by Bo98**
This is a core part of our CI now. We don't want the remote end changing under us randomly. Vendor it properly with CI testing.

### CODE REVIEW

File: .github/workflows/schemas.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+name: Update schema data
```
Comment (MikeMcQuaid) [inline .github/workflows/schemas.yml:1]: Thoughts on including this as part of one of the other "see if this file changes and if so commit/push it" workflows? There's a lot of duplicate code overlap between them.

File: Library/Homebrew/sbom.rb [hunk 2]
```diff
@@ -10,9 +10,7 @@
 # Rather than calling `new` directly, use one of the class methods like {SBOM.create}.
 class SBOM
   FILENAME = "sbom.spdx.json"
-  SCHEMA_URL = "https://spdx.github.io/spdx-3-model/model.jsonld"
-  SCHEMA_FILENAME = "sbom.spdx.schema.3.json"
-  SCHEMA_CACHE_TARGET = (HOMEBREW_CACHE/"sbom/#{SCHEMA_FILENAME}").freeze
+  SCHEMA_FILE = (HOMEBREW_LIBRARY_PATH/"data/schemas/sbom.spdx.schema.3.json").freeze
```
Comment (MikeMcQuaid) [inline Library/Homebrew/sbom.rb:13]: Do we expect to have more schemas coming here?

**Review by MikeMcQuaid — APPROVED** [review]
Thanks @Bo98! Good to go as-is to unbreak things but let's talk about review comments for a follow-up.

File: .github/workflows/schemas.yml (hunk 1 above)
Comment (Bo98) [reply .github/workflows/schemas.yml]: I think so too - I've made a mistake here anyway so have dropped this temporarily.

File: Library/Homebrew/sbom.rb (hunk 2 above)
Comment (Bo98) [reply Library/Homebrew/sbom.rb:13]: I think it would be a good idea to add the OCI ones here.

**Review by carlocab — APPROVED** [review]

**Comment by Bo98** [comment]
Turns out validation has never really worked properly. Will merge this as-is to unblock things as it maintains the status quo but need to investigate this much further.

File: Library/Homebrew/sbom.rb (hunk 2 above)
Comment (MikeMcQuaid) [reply Library/Homebrew/sbom.rb:13]: @Bo98 Cool, thanks.

**Comment by MikeMcQuaid** [comment]
Thanks @Bo98!

**Pull request merged**